eng/pipelines/templates/steps/sync-directory.yml

```diff
@@ -54,6 +54,22 @@ steps:
           -PushArgs "${{ parameters.PushArgs }}"
           -SkipCommit $${{parameters.SkipCheckingForChanges}}
 
+    - task: PowerShell@2
+      displayName: Push changes for Queueing Test Pipeline
+      condition: and(succeeded(), ne(variables['${{repo}}-template-definition-id'], ''))
+      inputs:
+        pwsh: true
+        workingDirectory: $(System.DefaultWorkingDirectory)/${{ repo }}
+        filePath: ${{ parameters.ScriptDirectory }}/git-branch-push.ps1
+        arguments: >
+          -PRBranchName "${{ parameters.UpstreamBranchName }}-ForTestPipeline"
+          -CommitMsg "${{ parameters.CommitMessage }}"
+          -GitUrl "https://$(azuresdk-github-pat)@github.com/${{ parameters.PROwner }}/${{ repo }}.git"
+          -PushArgs "${{ parameters.PushArgs }}"
+          -RemoteName "azure-sdk-test-fork"
+          -SkipCommit $${{parameters.SkipCheckingForChanges}}
```

Comment:
If we want to just do an amend we shouldn't pass SkipCommit as it is a little mis-leading